.circleci/config.yml

```diff
@@ -59,10 +59,10 @@ jobs:
             command: python3 -m pip install --no-cache-dir git+https://github.com/RedisLabsModules/RLTest.git@master
         - run:
             name: Tests
-            command: make tests
+            command: OS=rhel OS_VER=7 make tests
```

Comment:
Longer term - maybe use readies since it can do this for us (effectively). Today - no need.

Reply:
readies returns centos while I need rhel. This is why I did it this way.